cw-orch-daemon/src/core.rs

```diff
@@ -220,37 +218,6 @@ impl DaemonAsync {
         })
     }
 
-    /// Upload a contract to the chain.
-    pub async fn upload(
```

Comment:
DaemonAsync can't implement `TxHandler` because the async fns... hence why these were implemented manually. Why are we removing `upload` ?

Reply:
I guess DaemonAsync::upload was unused: https://github.com/AbstractSDK/cw-orchestrator/blob/36ff755de436ac8d37da52ebba550c1e62f0357f/cw-orch-daemon/src/sync/core.rs#L120-L123

Reply:
Got it back, now it uses upload directly from DaemonAsync as well!